appyters/ATC_Code_Drugmonizome_ETL/ATC_codes.ipynb

```diff
@@ -0,0 +1,488 @@
+    "\n",
+    "#### Data source : https://www.drugbank.ca/releases/latest"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# appyter init\n",
+    "from appyter import magic\n",
+    "magic.init(lambda _=globals: _())"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "import sys\n",
+    "import datetime\n",
+    "import zipfile\n",
+    "import collections\n",
+    "import xml.etree.ElementTree as ET\n",
+    "\n",
+    "import pandas as pd\n",
+    "import numpy as np\n",
+    "import drugmonizome.utility_functions as uf"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%load_ext autoreload\n",
+    "%autoreload 2"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Notebook Information"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "print('This notebook was run on:', datetime.date.today(), '\\nPython version:', sys.version)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Create Output Path"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "output_name = 'atc_codes'\n",
+    "path = 'output/drugmonizome_atc_codes'\n",
+    "if not os.path.exists(path):\n",
+    "    os.makedirs(path)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Initializing Notebook"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%appyter hide_code\n",
+    "\n",
+    "{% do SectionField(\n",
+    "    name='data',\n",
+    "    title='Upload Data',\n",
+    "    img='load_icon.png'\n",
+    ") %}"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%appyter code_eval\n",
+    "\n",
+    "{% do DescriptionField(\n",
+    "    name='description',\n",
+    "    text='The example below was sourced from <a href=\"https://www.drugbank.ca/releases/latest\" target=\"_blank\">drugbank.ca</a>. If clicking on the example does not work, it should be downloaded directly from the source website.',\n",
+    "    section='data'\n",
+    ") %}\n",
+    "\n",
+    "{% set xml_file = FileField(\n",
+    "    constraint='.*\\.xml.zip$',\n",
+    "    name='atc codes', \n",
+    "    label='ATC Codes (xml.zip)', \n",
+    "    default='drugbank_all_full_database.xml.zip',\n",
+    "    examples={\n",
+    "        'drugbank_all_full_database.xml.zip': 'https://www.drugbank.ca/releases/5-1-7/downloads/all-full-database'\n",
```

Comment:
I have the url path you referenced set within the example parameter of the FileField module, is there anything else I should have done?

Reply:
@ekropiwnicki no this is a bug in the validator, I will fix it and merge your PR thanks